docs/tutorials/kicad.md

```diff
@@ -0,0 +1,150 @@
+---
+sidebar_position: 1
+---
+
+# Kicad setup
+In this tutorial you will learn how to setup your computer for Kicad and create electric schematics.
+
+## Resources
+
+1. [Kicad Introduction](https://docs.kicad.org/8.0/en/introduction/introduction.html)
+2. [Getting started in Kicad](https://docs.kicad.org/8.0/en/getting_started_in_kicad/getting_started_in_kicad.html)
+
+
+## What is Kicad?
+
+KiCad is a free and open-source electronics design automation (EDA) suite. It features schematic capture, integrated circuit simulation, printed circuit board (PCB) layout, 3D rendering, and plotting/data export to numerous formats. KiCad also includes a high-quality component library featuring thousands of symbols, footprints, and 3D models. KiCad has minimal system requirements and runs on Linux, Windows, and macOS.
+
+## Installation 
+
+You can get Kicad from [here](https://www.kicad.org/download/) for your OS.
+
+## How to create a new project
+
+The first thing you do when starting a new design is to create a new project. Open Kicad, click `File`->`New Project`, browse to your desired location and give your project a name. 
+
+<div align="center">
+![New project](images/new_project.png)
+</div>
+
+There is a project file with the extension `.kicad_pro`, a schematic file with the extension `.kicad_sch`, and a board file with the extension `.kicad_pcb`. These files all share a name with your project.
+
+:::note
+Before creating a new project go through the next steps to make upcoming changes take effect.
+:::
+
+## How to add custom schematics, footprints and 3D models
+
+Kicad comes with a standard library that has a wide variety of components. For most project you will only need to import just a bunch of specific integrated circuits or custom components. This semester we will work with Raspberry Pi Pico W, a development board designed around RP2040 microcontroller(MCU). Kicad standard library has the RP2040 designs, but for the laboratory work we need to import the development board design files.
```

Comment:
```suggestion
Kicad comes with a standard library that has a wide variety of components. For most project you will only need to import just a bunch of specific integrated circuits or custom components. We will work with a *Raspberry Pi Pico W*, a development board designed around RP2040 microcontroller(MCU). KiCad's standard library has the RP2040 designs, but for the lab work we need to import the development board design files.
```

Reply:
*but for the lab work we need to import the development board design files*

Please explain why this is needed.